Makefile

```diff
@@ -18,17 +28,17 @@ test:
 .PHONY: run-integration-tests
 run-integration-tests: export DOCKER_HOSTNAME?=localhost:5000
 run-integration-tests: export DOCKER_NAMESPACE?=m4d-system
-run-integration-tests: export VALUES_FILE=m4d/integration-tests.values.yaml
+run-integration-tests: export VALUES_FILE=charts/m4d/integration-tests.values.yaml
 run-integration-tests:
 	$(MAKE) kind
-	$(MAKE) -C charts vault
-	$(MAKE) -C charts wait-for-vault
-	$(MAKE) -C charts cert-manager
+	$(MAKE) -C third_party/vault deploy
+	$(MAKE) -C third_party/vault wait-for-vault
+	$(MAKE) -C third_party/cert-manager deploy
 	$(MAKE) -C third_party/datashim deploy
 	$(MAKE) docker
```

Comment:
Consider removing `docker` target and using here `$(MAKE) docker-build docker-push` directly